### ISSUE TITLE
useQuery stuck indefinitely in ssr when skipCache is set to true

### ISSUE THREAD

**Issue by bmamouri** [nearform/graphql-hooks#396]
### Package
`graphql-hooks`
`graphql-hooks-ssr`

### Environment

- `graphql-hooks` version: 4.0.2
- `graphql-hooks-ssr` version: 1.1.4
- `graphql-hooks-memcache` version: 1.2.4
- `react` version: 16.11.0
- Browser: Chrome Version 78.0.3904.87

### Description

In `useQuery` if `skipCache` is set, the query run indefinitely without raising any error or returning any result. Setting `skipCache` to false would resolve the issue.

I have to skip the cache because it is not yet possible to update the cache after a mutation: https://github.com/nearform/graphql-hooks/issues/52

### How to reproduce

The following query run indefinitely when executed on the server

```jsx
    const {error, data, refetch} = useQuery(QUERY, {skipCache: true});
```

**Comment by bmullan91** [comment]
@bmamouri You could update the cache manually, it just requires a bit of code than it should be 😅 

```js
import React, { useContext } from 'react'
import { ClientContext, useMutation } from 'graphql-hooks'
import { LIST_USERS_QUERY } from './somewhere'

const CREATE_USER_MUTATION = `...`

function CreateNewUser() {
  const client = useContext(ClientContext)
  const [createUser] = useMutation(CREATE_USER_MUTATION)

  const createUserAndUpdateCache = async (...args) => {
    const { data } = await createUser(...args)

    const cacheKey = client.getCacheKey({ query: LIST_USERS_QUERY })
    const existingResult = client.cache.get(cacheKey)
    client.cache.set(cacheKey, {
      ...existingResult,
      data: {
        users: [...existingResult.data.users, data.createUser]
      }
    })
  }
}


// use createUserAndUpdateCache in your event handler
```

**Comment by bmullan91** [comment]
Available in `graphql-hooks@4.1.1` 🚀
